src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int capacity = DEFAULT_INITIAL_CAPACITY;
+    private int size;
+    private int threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
+
+    private int hash(Object key) {
```

Comment:
```suggestion
    private int getIndex(Object key) {
```
тут не тільки хешується ключ а і одразу шукається індекс бакета по хешу